### DISCUSSION TITLE
Configure Wifi Connection As Well

### DISCUSSION THREAD

**Discussion by dommy1C** [RaspAP/raspap-webgui#758] [Q&A]
Hi
Obviously great work here.
Is it possible to add a wifi connection so that the Rpi can connect to the internet as well.
Sorry if stupid question.  Newbie with all of this.
I can see the wifi I want to use in the client settings but cannot add it for some reason.
If I go to HotSpot | Advanced and try to select WiFi client AP mode (to see if that is what I need to do to enable wifi) I cannot because it is disabled?  I am not sure if this is a bug so will raise it there as well.

**Comment by arturi** [comment]
The note on the install guide https://raspap.com#simultaneous-ap-and-wifi-client says:

> Note: This option is disabled until you configure your system as a wireless client. For a device operating in managed mode without an eth0 connection, this configuration must be enabled before a reboot.

Maybe you could try connecting to a WiFi network first, then go back to the Hotspot / Advanced screen.
